pkgs/by-name/du/dummyhttp/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, darwin
```

```suggestion
, darwin
, stdenv
```

It may be necessary for reasons discussed below.